src/GraphQL/DataObjectInputProcessor/ManyToManyRelation.php

```diff
@@ -45,6 +46,10 @@ public function process($object, $newValue, $args, $context, ResolveInfo $info)
 
                     if ($element) {
                         $result[] = $element;
+                    } else {
+                        throw new NotFoundException(
+                            "Doesn't found element: " . $newValueItemValue['fullpath']
```

Comment:
Looks like grammatically incorrect and only covering `fullpath`, should be more like `Element with id or path not found`, see also [getElementByTypeAndIdOrPath](https://github.com/pimcore/data-hub/blob/3a454f2ac097e3d4b2e99a5eba92e4e087d72b61/src/GraphQL/Traits/ElementIdentificationTrait.php#L39-L65) on possible errors, since it is what returned to `$element` on https://github.com/pimcore/data-hub/blob/b76286cc0ade02e027eafe4a9e33c9e735ada7bf/src/GraphQL/DataObjectInputProcessor/ManyToManyRelation.php#L45